README.md

```diff
@@ -1,25 +1,78 @@
 
-Formulas for bootstrapping and managing a high availability cluster
-using crmsh.
+Salt formula to bootstrap and manage a [ClusterLabs](https://clusterlabs.org/) high availability cluster.
 
-Mainly adapted to SUSE / openSUSE Linux distributions, but should be
-usable on other distributions with minor modifications.
+Mainly adapted to SUSE / openSUSE Linux distributions (it is based in
+[crmsh](https://github.com/ClusterLabs/crmsh)), but it should be usable on other distributions with
+some modifications.
+
+## Features
+
+The formula provides the capability to create and configure a multi node HA cluster. Here some
+features:
+- Initialize a cluster
+- Join a node to an existing cluster
+- Remove a node from an existing cluster
+- Configure the pre-requirements (install required packages, configure `ntp/chrony`, create ssh-keys, etc)
+- Auto detect if the cluster is running in a cloud provider (Azure, AWS or GCP)
+- Configure sbd
+- Configure corosync
+- Configure the resource agents
+- Install and configure the [ha_cluster_exporter](https://github.com/ClusterLabs/ha_cluster_exporter)
+
+## Installation
+
+The project can be installed in many ways, including but not limited to:
+
+1. [RPM](#rpm)
+2. [Manual clone](#manual-clone)
+
+### RPM
+
+On openSUSE or SUSE Linux Enterprise you can just use the `zypper` system package manager:
+```shell
+zypper install habootstrap-formula
+```
+
+**Important!** This will install the formula in `/usr/share/salt-formulas/states/cluster`. Make sure that `/usr/share/salt-formulas/states` entry is correctly configured in your salt minion configuration `file_roots` entry if the formula is used in a masterless mode.
+
+You can find the latest development repositories at [SUSE's Open Build Service](https://build.opensuse.org/package/show/network:ha-clustering:sap-deployments:devel/habootstrap-formula).
+
+### Manual clone
+
+```
+git clone https://github.com/SUSE/habootstrap-formula
+cp -R cluster /srv/salt
+```
+
+**Important!** The formulas depends on `salt-shaptools` package, so make sure it is installed properly if you follow the manual installation.
 
 ## Usage
 
-To use, configure the cluster options using pillar data as described
-in `pillar.example`. To create multiple clusters, different pillar
-data can be applied to different groups of nodes with different nodes
-as the init node.
+To use the formula the `cluster` entry must be included in the salt execution `top.sls` file. Here an example to execute the cluster formula in all of the nodes:
+
+```
+# This file is /srv/salt/top.sls
+base:
+  '*':
+    - cluster
+```
+
+To configure the execution a pillar file is needed. Here an example of a pillar file for this formula: [pillar.example](https://github.com/SUSE/habootstrap-formula/blob/master/pillar.example)
+This file must be stored in `/srv/pillar` as `cluster.sls` and the same folder must contain a `top.sls` to use it. For example:
+
+```
+# This file is /srv/pillar/top.sls
```

Comment:
Shouldn't that be `/srv/pillar//cluster.sls`? Some lines above you have `/srv/pillar/top.sls`.

Reply:
No, we need to have all of these files:
```/srv/salt/top.sls
/srv/pillar/top.sls
/srv/pillar/cluster.sls
```